cpg_workflows/stages/large_cohort.py

```diff
@@ -31,7 +31,15 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
 
         j = get_batch().new_job('Combiner', (self.get_job_attrs() or {}) | {'tool': 'hail query'})
 
-        init_batch_args = {'worker_memory': 'highmem'} if get_config()['workflow'].get('highmem_workers') else None
+        init_batch_args = {}
+        config = get_config()['workflow']
+
+        if config.get('highmem_workers'):
+            init_batch_args['worker_memory'] = 'highmem'
+        if config.get('highmem_drivers'):
+            init_batch_args['driver_memory'] = 'highmem'
+        if 'driver_cores' in config:
+            init_batch_args['driver_cores'] = config['driver_cores']
 
```

Comment:
Could we just add a logging.warning message here if none of these values are specified, so the user knows to ensure they're included in the config if they're getting OOM errors.